src/Sulu/Component/Content/Types/Link.php

```diff
@@ -112,6 +117,17 @@ public function getContentData(PropertyInterface $property): ?string
             $url = \sprintf('%s#%s', $url, $value['anchor']);
         }
 
+        $mediaReferenceStore = $this->getMediaReferenceStore($value['provider']);
```

Comment:
Rename to `referenceStore` as we do not know exactly for which entity the referenceStore is fetched from the pool. The `value['provider']` defines this.